python/integration_tests/test_upkeep_expiry.py

```diff
@@ -231,11 +231,11 @@ def test_upkeep_expiry() -> None:
         for i in range(num_messages):
             if i % 2 == 0:
                 task_activation = generate_task_activation(
-                    OnAttemptsExceeded.ON_ATTEMPTS_EXCEEDED_DISCARD, 3
+                    OnAttemptsExceeded.ON_ATTEMPTS_EXCEEDED_DISCARD, 15
```

Comment:
Why did you change these?

Reply:
The integration test was flakey. Tasks were expiring before all tasks were even inserted into sqlite which was failing this test.